src/vcpkg.json

```diff
@@ -1,120 +1,124 @@
 {
-    "name": "wazuh-agent",
-    "version": "5.0.0",
-    "dependencies": [
-      {
-        "name": "boost-asio",
-        "version>=": "1.85.0"
-      },
-      {
-        "name": "boost-beast",
-        "version>=": "1.85.0"
-      },
-      {
-        "name": "boost-uuid",
-        "version>=": "1.85.0"
-      },
-      {
-        "name": "boost-url",
-        "version>=": "1.85.0"
-      },
-      {
-        "name": "cjson",
-        "version>=": "1.7.17"
-      },
-      {
-        "name": "curl",
-        "version>=": "8.5.0"
-      },
-      {
-        "name": "fmt",
-        "version>=": "10.2.1"
-      },
-      {
-        "name": "gtest",
-        "version>=": "1.15.2"
-      },
-      {
-        "name": "jwt-cpp",
-        "version>=": "0.7.0"
-      },
-      {
-        "name": "libarchive",
-        "version>=": "3.7.5"
-      },
-      {
-        "name": "libdb",
-        "version>=": "18.1.40",
-        "platform": "linux"
-      },
-      {
-        "name": "libplist",
-        "version>=": "2023-06-15#1",
-        "platform": "osx"
-      },
-      {
-        "name": "libpopt",
-        "version>=": "1.16#17"
-      },
-      {
-        "name": "librpm",
-        "version>=": "4.18.2",
-        "platform": "linux"
-      },
-      {
-        "name": "lua"
-      },
-      {
-        "name": "nlohmann-json",
-        "version>=": "3.11.3"
-      },
-      {
-        "name": "openssl",
-        "version>=": "3.3.2"
-      },
-      {
-        "name": "procps",
-        "version>=": "3.3.0",
-        "platform": "linux"
-      },
-      {
-        "name": "spdlog",
-        "version>=": "1.14.0"
-      },
-      {
-        "name": "sqlite3",
-        "version>=": "3.45.0#0"
-      },
-      {
-        "name": "sqlitecpp",
-        "version>=": "3.3.2"
-      },
-      {
-        "name": "yaml-cpp",
-        "version>=": "0.8.0"
-      },
-      {
-        "name": "zlib",
-        "version>=": "1.3.1"
-      }
-    ],
-    "overrides": [
-      {
-        "name": "lua",
-        "version": "5.4.1"
-      }
-    ],
-    "vcpkg-configuration": {
-      "default-registry": {
-        "kind": "git",
-        "repository": "https://github.com/Microsoft/vcpkg",
-        "baseline": "f7423ee180c4b7f40d43402c2feb3859161ef625"
-      },
-      "overlay-ports": [
-        "./ports-overlay/jwt-cpp",
-        "./ports-overlay/procps",
-        "./ports-overlay/libdb",
-        "./ports-overlay/librpm"
-      ]
+  "name": "wazuh-agent",
```

Comment:
There's some unnecessary re-formatting of this file, it should only have the change related to adding boost options.